src/locales/pl.ts

```diff
@@ -171,17 +171,18 @@ export default {
     clare_of_assisi_virgin: 'Św. Klary, dziewicy',
     clement_i_pope: 'Św. Klemensa I, papieża i męczennika',
     columba_of_iona_abbot: 'Św. Kolumbana, zakonnika',
-    columba_of_iona_abbot_copatron_of_ireland: 'Św. Kolumbana, zakonnika i patrona Irlandii', // TODO: How to say `secundary patron` in Polish?
+    columba_of_iona_abbot_copatron_of_ireland: 'Św. Kolumbana, zakonnika, patrona Irlandii',
     conversion_of_saint_paul_the_apostle: 'Nawrócenie św. Pawła, Apostoła',
     cornelius_i_pope_and_cyprian_of_carthage_bishop_martyrs:
       'Świętych męczenników Korneliusza, papieża, i Cypriana, biskupa',
     cosmas_of_cilicia_and_damian_of_cilicia_martyrs: 'Świętych męczenników Kosmy i Damiana',
     cyril_of_alexandria_bishop: 'Św. Cyryla Aleksandryjskiego, biskupa i doktora Kościoła',
     cyril_of_jerusalem_bishop: 'Św. Cyryla Jerozolimskiego, biskupa i doktora Kościoła',
     cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop: 'Świętych Cyryla, mnicha i Metodego, biskupa',
-    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe:
-      'Świętych Cyryla, mnicha i Metodego, biskupa, patronów Europy', // TODO: How to say `secundary patrons` in Polish?
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe:    
```

Comment:
Could you remove the trailing whitespace please? Even better: could you install and use [`.editorconfig`](editorconfig.org) please? :smiley: